### DISCUSSION TITLE
How to block autoplay for all website?

### DISCUSSION THREAD

**Discussion by radiorambo** [Eloston/ungoogled-chromium#2509] [Q&A] [answered]
I want to block all the website from autoplay. I checked site setting but couldn't find.

**Comment by PF4Public** [comment, marked as answer]
There is no such setting in ungoogled-chromium

**Reply by tomasz1986** [reply]
There used to be a flag in Chromium for this at chrome://flags/#autoplay-policy. It is long gone though. I guess there is no simple way to restore this option, right?

There are some extensions or userscripts with this kind of functionality, but they are usually limited to specific sites only (like YouTube, etc.).

**Reply by PF4Public** [reply]
It might be available via [Enterprise policies](https://chromium.googlesource.com/chromium/src/+/HEAD/docs/enterprise/policies.md)
